src/nvim/indent_c.h

```diff
@@ -3,6 +3,12 @@
 
 #include "nvim/vim.h"
 
+// Find result cache for cpp_baseclass
+typedef struct {
+    int found;
+    lpos_T lpos;
+} cpp_baseclass_cache_T;
```

Comment:
I tried moving the struct to `indent_c.c`, but I keep getting this:

    [11/50] Building C object src/nvim/CMakeFiles/nvim.dir/indent_c.c.o
    FAILED: /usr/lib/colorgcc/bin/cc  -DHAVE_CONFIG_H -DINCLUDE_GENERATED_DECLARATIONS -Iconfig -I../src -isystem ../.deps/usr/include -isystem ../.deps/usr/include/luajit-2.0 -Isrc/nvim/auto -Iinclude -Wconversion -g   -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiagnostics-color=auto -MMD -MT src/nvim/CMakeFiles/nvim.dir/indent_c.c.o -MF src/nvim/CMakeFiles/nvim.dir/indent_c.c.o.d -o src/nvim/CMakeFiles/nvim.dir/indent_c.c.o   -c ../src/nvim/indent_c.c
    In file included from ../src/nvim/indent_c.c:23:0:
    src/nvim/auto/indent_c.c.generated.h:36:33: error: unknown type name ‘cpp_baseclass_cache_T’
     static int cin_is_cpp_baseclass(cpp_baseclass_cache_T *cached);

I don't quite understand how the header auto generation works so I don't know how to fix it.

Here's a branch where I moved the struct to `indent_c.c` https://github.com/johanhelsing/neovim/tree/vim-7.4.803-move-struct